openedx/core/djangoapps/dark_lang/tests.py

```diff
@@ -253,77 +256,32 @@ def _post_clear_preview_lang(self):
         """
         return self.client.post('/update_lang/', {'action': 'reset_preview_language'})
 
-    def _set_client_session_language(self, session_language):
-        """
-        Set the session language in the Client
-        """
-        session = self.client.session
-        session[LANGUAGE_SESSION_KEY] = session_language
-        session.save()
-
     def test_preview_lang_with_released_language(self):
+        self._set_client_cookie_language('rel')
```

Comment:
Can you clarify the transition from the old to new test? 
1. Why is this needed up front, but wasn't for the old test? 
2. Why did the old test use `self._set_client_session_language('notrel')` later, but this one doesn't?

Reply:
I have reverted this change.